gnuradio-runtime/lib/block_executor.cc

```diff
@@ -146,7 +147,6 @@ namespace gr {
         }
       }
       else  {
-        std::cerr << "Error: block_executor: propagation_policy 'ONE-TO-ONE' requires ninputs == noutputs" << std::endl;
```

Comment:
At first, I changed this to `GR_LOG_ERROR`. But soon found that function is not a class function. So, can't access loggers.